SRC/cgeqp3.f

```diff
@@ -249,6 +249,11 @@ SUBROUTINE CGEQP3( M, N, A, LDA, JPVT, TAU, WORK, LWORK, RWORK,
             JPVT( J ) = J
          END IF
    10 CONTINUE
+*
+*     Quick return if possible.
+*
+      IF( M.EQ.0 ) RETURN
```

Comment:
Why not use  `MINMN.EQ.0`?

Reply:
I used `M.EQ.0` because I was trying to change the code as little as possible. 
`MINMN.EQ.0` would be equivalent to `M.EQ.0 .OR. N.EQ.0`.
The N=0 case appears to work with the existing code (NXFD is 0 when N=0 and most code is skipped).

But I have no objection to replacing `M.EQ.0` with `MINMN.EQ.0`.
If this additional change is required, what would be the best way to accomplish this?
(Please forgive my ignorance, I am not very familiar with git.)

Could the PR be accepted in it's current form in order to pick up the work space fix and since it does function correctly as it is now? I would be happy to submit a second PR which changes `M.EQ.0` to `MINMN.EQ.0` in all 4 xGEQP3 files. Is that acceptable? (I am afraid I don't know how to update the contents of an open PR, but I can try to find out, if needed.)

Reply:
Okay, perhaps I would only need to change `M.EQ.0` to `MINMN.EQ.0` and do a second 
git commit ...
from the same branch? Do you want me to try that?